bot/exts/evergreen/status_codes.py

```diff
@@ -1,25 +1,30 @@
 from http import HTTPStatus
+from random import choice
 
 import discord
 from discord.ext import commands
 
-from bot.utils.extensions import invoke_help_command
-
 HTTP_DOG_URL = "https://httpstatusdogs.com/img/{code}.jpg"
 HTTP_CAT_URL = "https://http.cat/{code}.jpg"
 
 
 class HTTPStatusCodes(commands.Cog):
-    """Commands that give HTTP statuses described and visualized by cats and dogs."""
+    """
+    Fetch an image depicting HTTP status codes as a dog or a cat.
+
+    If neither animal is selected a cat or dog is chosen randomly for the given status code.
+    """
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    @commands.group(name="http_status", aliases=("status", "httpstatus"))
-    async def http_status_group(self, ctx: commands.Context) -> None:
-        """Group containing dog and cat http status code commands."""
-        if not ctx.invoked_subcommand:
-            await invoke_help_command(ctx)
+    @commands.group(name="http_status", aliases=("status", "httpstatus"), invoke_without_command=True)
+    async def http_status_group(self, ctx: commands.Context, code: int) -> None:
+        """Choose a cat or dog randomly for the given status code."""
+        subcmd = choice((self.http_cat, self.http_dog))
+
+        if await subcmd.can_run(ctx):
```

Comment:
Everything looks good 

except for the
`command,can_run()` if statement., if the user can the `group` main command, then the subcommands should have no problem running. We don't have any checks on the functions too. The invalid argument error in any way
caught in the error handler. But I guess it doesn't hurt to have it.

Reply:
I was told to use this instead of \_\_call__ so that check would run.

Reply:
Right, but this group command has the same checks as the subcommands (which is the global check) so there isn't really any point in calling `can_run`.